build.sbt

```diff
@@ -5,7 +5,7 @@ import scala.sys.process._
 
 enablePlugins(PackPlugin)
 
-val chiselVersion = "3.5-SNAPSHOT"
+val chiselVersion = "3.5.0"
```

Comment:
FWIW, main is currently using 3.5.2 and this would revert that.

Reply:
Oh, I guess with the way the commit history works, this just ends up as a no-change so never mind me here.

Reply:
I think you're commenting on a commit, I was basically reverting a prior commit so that the overall PR makes no changes to `build.sbt`.